aptos-move/block-executor/src/view.rs

```diff
@@ -1035,7 +1054,10 @@ impl<'a, T: Transaction, S: TStateView<Key = T::Key>, X: Executable> LatestView<
                             err
                         );
                         self.mark_incorrect_use();
-                        return Err(err);
+                        return Err(PartialVMError::new(
+                            StatusCode::UNKNOWN_INVARIANT_VIOLATION_ERROR,
```

Comment:
why is this not a DELAYED_FIELDS_CODE_INVARIANT_ERROR and we go to fallback? that is what mark_incorrect_use would do, if you don't return UNKNOWN_INVARIANT_VIOLATION_ERROR

Reply:
Good catch, right, changed to DELAYED_FIELDS_CODE_INVARIANT_ERROR.